src/manifest.prod.json

```diff
@@ -6,7 +6,7 @@
   "manifest_version": 2,
   "minimum_chrome_version": "79",
   "homepage_url": "https://www.analogue.app",
-  "permissions": ["activeTab", "contextMenus", "notifications", "*://*.analogue.app/*"],
+  "permissions": ["activeTab", "contextMenus", "notifications", "*://*.analogue.app/*", "<all_urls>"],
```

Comment:
@spencerjsmall do we need `<all_urls>` here? Assuming it's because of the keyboard shortcut command. If so, we can remove the `"*://*.analogue.app/*"` and localhost one in manifest.dev since all_urls will cover this.